odoo/upgrade_code/17.5-00-tree-to-list.py

```diff
@@ -0,0 +1,37 @@
+import re
+
+
+def upgrade(file_manager):
+    nb = len([1 for file in file_manager if file.path.name.rsplit('.', 1)[1] in ('xml', 'js', 'py')])
```

Comment:
`nb` does not mean anything, except it's a number.  At least `total` suggests to total amount:
```suggestion
    total = len([1 for file in file_manager if file.path.suffix in ('xml', 'js', 'py')])
    if not total:
        return  # nothing to do!
```